common/src/dcd/common/constants.d

```diff
@@ -80,6 +81,12 @@ immutable ConstantCompletion[] predefinedVersions = [
 	ConstantCompletion("CRuntime_Microsoft"),
 	ConstantCompletion("CRuntime_Musl"),
 	ConstantCompletion("CRuntime_UClibc"),
+	ConstantCompletion("CRuntime_WASI"),
+	ConstantCompletion("CppRuntime_Clang"),
+	ConstantCompletion("CppRuntime_DigitalMars"),
+	ConstantCompletion("CppRuntime_Gcc"),
+	ConstantCompletion("CppRuntime_Microsoft"),
+	ConstantCompletion("CppRuntime_Sun"),
```

Comment:
where do you have the `CppRuntime_*` values from?

Reply:
Hmm, these identifiers were already defined in dmd but not in the spec...

Reply:
updated! https://github.com/dlang/dlang.org/commit/3202e42906fbd141a0c9c1ac3fa1c1b5ec192db4